tests/node/streaming/video/VideoStreamingParametersTest.js

```diff
@@ -91,9 +91,10 @@ describe('VideoStreamingParametersTest', function () {
 
         format = new VideoStreamingFormat(VideoStreamingProtocol.RTP, VideoStreamingCodec.H264);
         capability.setSupportedFormats([format]);
+        capability.setPreferredFPS(15);
         params.update(capability);
         Validator.assertEquals(params.getFormat(), format);
 
```

Comment:
`setPreferredFPS` method added to the test but there's no test to check it got set with the update.
Needs `Validator.assertEquals(params.getFrameRate(), 15);`

Reply:
That makes sense. Updated accordingly,